### ISSUE TITLE
FR: Merge duplicate instructions on GCP onboarding, roles, service enablements, billing and bootstrap project - continuation of 1175

### ISSUE THREAD

**Issue by obriensystems** [terraform-google-modules/terraform-example-foundation#1204]
### TL;DR

assigned to https://github.com/fmichaelobrien
PR developed in fork https://github.com/GoogleCloudPlatform/pbmm-on-gcp-onboarding/issues/407

specifically the prerequisites sections with roles and service enablements duplicated across
"For the user who will run the procedures in this document, grant the following roles:"
and
"Enable the following additional services on your current bootstrap project"
in
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/README.md#prerequisites
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/README-Terraform-Cloud.md#requirements
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/README-GitLab.md#deploying-a-gitlab-cicd-compatible-environment
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/README-GitHub.md#requirements

no section in
- https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/README-Jenkins.md#overview

Check each terraform module specific to the build agent - for example
- https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/modules/jenkins-agent/README.md#Requirements

Fix:
detail all common roles and service enablements in the main readme, put links back to this readme from the extended readme's and add specific roles/services per gitlab/github/terraform-cloud/ado in extended readme's

see
- https://github.com/GoogleCloudPlatform/pbmm-on-gcp-onboarding/pull/363
- https://github.com/GoogleCloudPlatform/pbmm-on-gcp-onboarding/issues/362
- https://github.com/terraform-google-modules/terraform-example-foundation/pull/1175
- https://github.com/terraform-google-modules/terraform-example-foundation/issues/1133
